src/index.ts

```diff
@@ -117,37 +150,54 @@ const initialize = (apiKey: string, target: Target, options: Options): Result =>
           logError(error)
         })
         .finally(() => {
-          metricsSchedulerId = setTimeout(scheduleSendingMetrics, METRICS_FLUSH_INTERVAL)
+          metricsSchedulerId = window.setTimeout(scheduleSendingMetrics, METRICS_FLUSH_INTERVAL)
```

Comment:
Just curious... this file uses both `globalThis` and `window`? Is there any reason to have gone with `window` here?